docs/agented-vm-guide.md

```diff
@@ -0,0 +1,267 @@
+              cloud-assigned-name: ubuntu2004
+              cloud-assigned-vpc-id: vpc-0d6bb6a4a880bd9ad
+API Version:  crd.cloud.antrea.io/v1alpha1
+Kind:         VirtualMachine
+Metadata:
+  Creation Timestamp:  2022-09-27T12:18:23Z
+  Generation:          1
+  Managed Fields:
+    API Version:  crd.cloud.antrea.io/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .:
+          f:cloud-assigned-id:
+          f:cloud-assigned-name:
+          f:cloud-assigned-vpc-id:
+        f:ownerReferences:
+          .:
+          k:{"uid":"42467d8c-9ae7-433b-879c-fdd7b2b843e1"}:
+    Manager:      nephe-controller
+    Operation:    Update
+    Time:         2022-09-27T12:18:23Z
+    API Version:  crd.cloud.antrea.io/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:agented:
+        f:networkInterfaces:
+        f:provider:
+        f:state:
+        f:tags:
+          .:
+          f:Name:
+        f:virtualPrivateCloud:
+    Manager:      nephe-controller
+    Operation:    Update
+    Subresource:  status
+    Time:         2022-09-27T12:18:23Z
+  Owner References:
+    API Version:           crd.cloud.antrea.io/v1alpha1
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  CloudEntitySelector
+    Name:                  cloudentityselector-aws-sample
+    UID:                   42467d8c-9ae7-433b-879c-fdd7b2b843e1
+  Resource Version:        7908985
+  UID:                     da9c4826-e9d6-43d7-87e0-44743f881fb1
+Status:
+  Agented:  true
+  Network Interfaces:
+    Ips:
+      Address:       10.0.1.45
+      Address Type:  InternalIP
+      Address:       54.193.202.222
+      Address Type:  ExternalIP
+    Mac:             02:96:fc:89:17:ed
+    Name:            eni-056b0d6da3f592943
+  Provider:          AWS
+  State:             running
+  Tags:
+    Name:                 ubuntu2004
+  Virtual Private Cloud:  vpc-0d6bb6a4a880bd9ad
+Events:                   <none>
+```
+
+## ExternalNode CR Creation
+
+The `Nephe Controller` will create an ExternalNode CR corresponding to each
+VirtualMachine CR, which has the `agented` flag enabled. In the below example,
+since VM `i-019459b33d951b62e` is configured with agented flag set to true,
+a corresponding ExternalNode `virtualmachine-i-019459b33d951b62e` is created.
+
+```bash
+kubectl get en -A
+```
+
+```text
+# Output
+NAMESPACE   NAME                                 AGE
+vm-ns       virtualmachine-i-019459b33d951b62e   6s
+```
+
+## ExternalEntities CR Creation
+
+For each ExternalNode CR, `antrea-controller` will create a corresponding
+ExternalEntity CR. In this example, the ExternalEntity
+`virtualmachine-i-019459b33d951b62e-5df40` corresponds to the ExternalNode
+`virtualmachine-i-019459b33d951b62e`, while the other three ExternalEntities
+are created by `nephe-controller` with ExternalNode field set to
+`nephe-controller`.
```

Comment:
remove this part.. its confusing to read

Reply:
Sure will remove the second part.